grype/db/v6/installation/curator.go

```diff
@@ -79,49 +80,46 @@ func NewCurator(cfg Config, downloader distribution.Client) (db.Curator, error)
 }
 
 func (c curator) Reader() (db.Reader, error) {
-	s, err := db.NewReader(
+	return db.NewReader(
 		db.Config{
 			DBDirPath: c.config.DBDirectoryPath(),
 			Debug:     c.config.Debug,
 		},
 	)
-	if err != nil {
-		return nil, err
-	}
-
-	m, err := s.GetDBMetadata()
-	if err != nil {
-		return nil, fmt.Errorf("unable to get vulnerability store metadata: %w", err)
-	}
-
-	_, err = c.validate(db.DescriptionFromMetadata(m), c.config.ValidateChecksum)
```

Comment:
This was causing the db to be hashed twice, since the `.Status()` call was also hashing the db and validating age and is required, I've removed the validation from this call altogether.